src/Buffer.mo

```diff
@@ -718,6 +732,17 @@ module {
   /// Returns true iff `buffer` contains `element` with respect to equality
   /// defined by `equal`.
   ///
+  ///
+  /// Example:
+  /// ```motoko include=initialize
+  ///
+  /// let buffer = Buffer.Buffer<Nat>(2);
+  /// buffer.add(2);
+  /// buffer.add(0);
+  /// buffer.add(3);
+  /// Buffer.contains<Nat>(buffer, 2, func (x, y) { x == y });
```

Comment:
```suggestion
  /// Buffer.contains<Nat>(buffer, 2, Nat.equals);
```
Most of the primitive types (such as `Nat`) have equality, comparison, and some form of hashing defined for them in the base library. So you generally want to pass those in instead of writing your own functions.

Reply:
sounds good. it was really for my own edification at first, since I struggled to create a lambda. I talked to Claudio though, and we should add all of the nuances around lambdas to Mikhail's "language tour of motoko"

Reply:
Ah okay that makes sense. 

And yeah definitely agreed. Lambdas and higher order functions are a big part of using Motoko